src/Pages/Shop/Cofee/Cofee.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import './Cofee.css';
+import Header from "../../../Components/Header/Header";
+import MenuBar from "../../../Components/MenuBar/MenuBar";
+import Footer from "../../../Components/Footer/Footer";
+import ShopBlock from "../../../Components/Block/ShopBlock";
+
+function Cofee(props) {
+    let shopblocks=[];
+    for(let i=0; i<props.dispatch.length; i++){
+        shopblocks.push(<ShopBlock dispatch={props.dispatch[i]}/>);
+    }
+    return (
+        <div className="cofee">
+            <Header />
+            <MenuBar  selected="cofee" />
+
+            <h1>Cofee</h1>
+            <div>
+                {shopblocks}
```

Comment:
we can use the method "map" for that.
{ dispatch.map((element) => (<ShopBlock dispatch={element}/>)) }